tests/src/Functional/BlockTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Drupal\Tests\localgov\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Functional tests for LocalGovDrupal install profile.
+ */
+class BlockTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'localgov_theme';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $profile = 'localgov';
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'localgov_core',
+    'localgov_services_landing',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->adminUser = $this->drupalCreateUser([
+      'access content',
+      'administer nodes',
+      'create localgov_services_landing content',
+      'edit own localgov_services_landing content',
+    ]);
+  }
+
+  /**
+   * Test blocks display.
+   */
+  public function testBlocksDisplay() {
+    $this->drupalLogin($this->adminUser);
+
+    // Check node title and summary display on a landing page.
+    $title = $this->randomMachineName(8);
+    $summary = $this->randomMachineName(16);
+    $body = $this->randomMachineName(32);
```

Comment:
This is fine.  An alternative could be Drupal\Component\Utility\Random::sentences()